src/Number/__tests__/Number.Test.js

```diff
@@ -29,4 +35,67 @@ describe('<Number />', () => {
         const wrapper = shallow(<Number {...numberProps} />);
         expect(isEqual(numberProps, wrapper.props())).toBe(true);
     });
+});
+
+describe('<Number /> mount', () => {
+    const onChangeCallback = sinon.spy();
+    const onBlurCallback = sinon.spy();
+
+    const wrapper = mount(
+        <Form name="numberTest">
```

Comment:
I'll fix this. In fact, I want to import this context from an external file if you're okay with that?